src/ILVerification/src/ILVerification.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
I think this can be netstandard1.3 - it was the first useful netstandard. (It is unusual for libraries to depend on netstandard1.5 - it does not have much extra compared to 1.3.)